From 40b6b57251212035626a7a2b9c2b27da72a87812 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Bj=C3=B8rn=20Forsman?= <bjorn.forsman@gmail.com>
Date: Mon, 5 May 2025 18:52:30 +0200
Subject: [PATCH] nut: 2.8.2 -> 2.8.3

https://github.com/networkupstools/nut/releases/tag/v2.8.3

Use the new --with-systemdsystempresetdir= option to avoid install error
where nut tries to install files to the systemd Nix store path.
---
 pkgs/by-name/nu/nut/package.nix | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/pkgs/by-name/nu/nut/package.nix b/pkgs/by-name/nu/nut/package.nix
index bdb6a0f792ef1e..6f5e774b74a7e2 100644
--- a/pkgs/by-name/nu/nut/package.nix
+++ b/pkgs/by-name/nu/nut/package.nix
@@ -25,11 +25,11 @@
 
 stdenv.mkDerivation rec {
   pname = "nut";
-  version = "2.8.2";
+  version = "2.8.3";
 
   src = fetchurl {
     url = "https://networkupstools.org/source/${lib.versions.majorMinor version}/${pname}-${version}.tar.gz";
-    sha256 = "sha256-5LSwy+fdObqQl75/fXh7sv/74132Tf9Ttf45PWWcWX0=";
+    sha256 = "sha256-1soX8LOQA7rHZJ6xerSnE+TV/KqP0a7cooNX1Z3wle0=";
   };
 
   patches = [
@@ -76,6 +76,7 @@ stdenv.mkDerivation rec {
     "--with-all"
     "--with-ssl"
     "--without-powerman" # Until we have it ...
+    "--with-systemdsystempresetdir=$(out)/lib/systemd/system-preset"
     "--with-systemdsystemunitdir=$(out)/lib/systemd/system"
     "--with-systemdshutdowndir=$(out)/lib/systemd/system-shutdown"
     "--with-systemdtmpfilesdir=$(out)/lib/tmpfiles.d"
